src/test/java/nextstep/subway/line/domain/SectionsTest.java

```diff
@@ -0,0 +1,150 @@
+package nextstep.subway.line.domain;
+
+import static nextstep.subway.exception.ExceptionMessage.*;
+import static org.assertj.core.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.List;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import nextstep.subway.station.domain.Station;
+
+class SectionsTest {
+
+    private Station 잠실역;
+    private Station 선릉역;
+    private Station 교대역;
+    private Station 당산역;
+    private Station 시청역;
+    private final Sections sections = new Sections();
+    Line line = new Line();
```

Comment:
```suggestion
    private Line line = new Line();
```

접근제어자를 생략한 것은 의도된 내용일까요?

Reply:
아닙니다 실수입니다 수정하겠습니다!